modules/src/Microsoft.Azure.IIoT.Modules.Discovery/src/Runtime/Config.cs

```diff
@@ -6,14 +6,16 @@
 namespace Microsoft.Azure.IIoT.Modules.Discovery.Runtime {
     using Microsoft.Azure.IIoT.Module.Framework;
     using Microsoft.Azure.IIoT.Module.Framework.Client;
+    using Microsoft.Azure.IIoT.OpcUa.Protocol;
+    using Microsoft.Azure.IIoT.OpcUa.Protocol.Runtime;
     using Microsoft.Azure.IIoT.Hub.Module.Client.Runtime;
     using Microsoft.Azure.IIoT.Diagnostics;
     using Microsoft.Extensions.Configuration;
 
     /// <summary>
     /// Wraps a configuration root
     /// </summary>
-    public class Config : DiagnosticsConfig, IModuleConfig {
+    public class Config : DiagnosticsConfig, IModuleConfig, IClientServicesConfig2{
```

Comment:
Why does discovery module require the client config.  Where is it used?

Reply:
this is necessary for the OPC stack application configuration setup. I remove the IClientServicesConfig that is no longer used